src/SamplesApp/SamplesApp.UITests/Windows_Storage/StorageFileTests/UnoSamples_Test_StorageFile_DateCreated.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_Storage.StorageFileTests
+{
+	[TestFixture]
+	class UnoSamples_Test_StorageFile_DateCreated : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.Android, Platform.iOS)]
+		public void StorageFile_DateCreated()
+		{
+			Run("UITests.Shared.Windows_Storage.StorageFileTests.StorageFile_DateCreated");
+
+			_app.WaitForElement(_app.Marked("sfdButtonCreate"));
+
+			TakeScreenshot("Initial");
+
+			var resultMessage = _app.Marked("sfdResult");
+
+			// step 1: create file
+			_app.Tap(_app.Marked("sfdButtonCreate"));
+
+			_app.WaitForText(resultMessage, "created");  // timeout here means: something impossible?
+
+			// step 2: test date just after create
+			_app.Tap(_app.Marked("sfdButtonTest"));
+			_app.WaitForText(resultMessage, "success");   // timeout here means: date is wrong
+			TakeScreenshot("AfterFirstCheck");
+
+			// step 3: test date several seconds later
+			_app.Wait(TimeSpan.FromSeconds(30));
```

Comment:
This is a big problem...  a test cannot take 30 seconds just for itself on the build server!

Is there a way to reduce this to max few seconds?

Specially when the result is not validated by the test itself...

Reply:
I don't know how many seconds would be enough to be sure that returned date is correct.
As if returned date is something like 'Date.Now', test should wait sufficiently long before getting date second time. But what means sufficiently long, I don't know :)
Maybe 10 seconds would be enough for test, and not locking build server for too long?

Reply:
We currently have more than 600 tests... imagine if all of them took 30 seconds... That would mean a 5 hours build!

You already have a test in the sample itself... it's not enough?

Reply:
What do you mean: "You already have a test in the sample itself"?

Test after some seconds is for checking that we get not current time, but time from creating file.
Of course, as this PR is only "a macro" for already implemented API, it probably could be skipped. But, you stated question: "Did you ensure it works on all supported platforms?", and this test is some way to get answer for this.

Reply:
tests are run sequentially?

Reply:
Yes, because they are run on an emulator, so it's not possible to have many tests run at the same time.

Reply:
For the wait time, you do not need to wait long. A few milliseconds should be enough, as long as it's different, and a greater date.

Reply:
I would convert test, and add DeleteAsync, DateCreated, and also StorageFolder.CreateAsync(string).

Few milliseconds is not good enough, as:
1. filetime resolution on underlying file system;
2. time spent in OS creating file could be longer than few milliseconds;

If time change (DST on/off) would occur between two reads, test would fail, but I hope that this would not be a frequent case :)

Reply:
I think build servers are running on UTC, a TZ without DST ;-)
Anyway, we can manage a failed build for those ;-)